src/lib/hash-input.unit-test.ts

```diff
@@ -1,148 +1,93 @@
 import {
```

Comment:
this test (which checks consistency of hash inputs between OCaml & JS) was refactored to a generative test